Open-Source/Contribute/Readme.md

```diff
@@ -22,7 +22,7 @@
 - [EddieBot](https://github.com/EddieHubCommunity/EddieBot)
 - [facebook-chat-api](https://github.com/Schmavery/facebook-chat-api)
 - [Falso.js](https://github.com/ngneat/falso)
-- [first-contributions](https://github.com/firstcontributions/first-contribution)
+- [first-contributions](https://firstcontributions.github.io)
```

Comment:
Please do not change the entire link. These two seems to be two separate repos

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable